src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -57,6 +57,11 @@
 public class ApexManifestFileReader {
 	private static Logger LOG = LoggerFactory.getLogger(ApexManifestFileReader.class);
 	public static ArrayList<String> nonExistantApexClassEntries = new ArrayList<String>();
+	private boolean includeTriggers;
+
+	public ApexManifestFileReader(boolean includeTriggers) {
```

Comment:
@pmedapuram : Agree about your stance on the purpose of boolean argument.. As discussed,  adding why querying the triggers needs to be skipped will  help